openedx/stanford/cms/djangoapps/contentstore/views/utilities/bulksettings.py

```diff
@@ -139,8 +139,7 @@ def utility_bulksettings_handler(request, course_key_string):
     """
 
     course_key = CourseKey.from_string(course_key_string)
-    response_format = request.REQUEST.get('format', 'html')
-
+    response_format = request.POST.get('format') or request.GET.get('format') or 'html'
```

Comment:
`request.REQUEST` was deprecated in Django 1.7 and then removed in Django 1.9 [1].

- [1] https://docs.djangoproject.com/en/1.8/ref/request-response/#django.http.HttpRequest.REQUEST